docs/metrics.md

```diff
@@ -201,6 +201,21 @@ Metrics for the [Four Golden Signals](https://sre.google/sre-book/monitoring-dis
     Some metric attributes may have high cardinality and are marked with ⚠️ to warn you. You may need to disable this metric or disable the attribute.
 <!-- titles should be the exact metric name for deep-linking, alphabetical ordered -->
 <!-- titles are without argo_workflows prefix -->
+#### `controller_build_info`
+
+The build information for this workflow controller
+
+| attribute   | explanation                                                            |
+|-------------|------------------------------------------------------------------------|
+| `version`   | The version of argo-workflows                                          |
+| `platform`  | Platform this is running on, as go describes it e.g. `linux/amd64` |
+| `gover`     | Version of go that built this workflow controller                  |
```

Comment:
also isn't the convention to use underscores? or is it different for attribute names vs metric names?

```suggestion
| `go_version`     | Version of Go used                  |
```

Reply:
Underscores are explicitly mentioned as word delimiters for attribute names in the convention docs: https://opentelemetry.io/docs/specs/semconv/general/attribute-naming/

So the second suggestion should be used.

It would also be be consistent with `argo version` which prints `GoVersion`

Reply:
Small follow-up: https://github.com/argoproj/argo-workflows/pull/13269#discussion_r1669064430